### DISCUSSION TITLE
[Question] Font for each artist/album

### DISCUSSION THREAD

**Discussion by GB-CG** [reupen/columns_ui#564] [Q&A]
Hi, can Columns UI display a different font for each artist/album in `Item Details` or `Playlist view`? I'm not super familiar with the more advanced syntax of FB2k. (Oops, also my tired brain completely ignored that the discussion tab exists. Sorry about that)

**Comment by reupen** [comment]
Hi,

The playlist view does not allow the font to be varied depending on the artist or album. (Colours can be varied, however.)

Item details does allow the font to be set dynamically using the $set_font() function. I'm not sure how you'd want to derive the font from the artist and album but you could combine $set_font() with [control-flow functions](https://wiki.hydrogenaud.io/index.php?title=Foobar2000:Title_Formatting_Reference#Control_flow_functions) to set the font based on some criteria. 

Also note that Item details displays a single track at a time.
